products/dns-security/docs/examples/request-domain-categorization-change.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: request-domain-categorization-change
+title: Request Domain Categorization Change
+description: DNS Security API example showing how to submit a change request for a domain's categorization.
+slug: /threat-vault/docs/examples/dns-security/request-domain-categorization-change
```

Comment:
Recommend you refrain from using slugs except for the intro docs (e.g. /threat-vault/api) The docs will always take the URL of the path + their ID. 

For example, this doc would be at pan.dev/dns-security/docs/examples/request-domain-categorization-change without the slug specified here. If you added slugs elsewhere that they aren't needed I'd recommend removing them